relays/messages-relay/src/message_race_loop.rs

```diff
@@ -61,36 +58,47 @@ type SourceClientState<P> = ClientState<<P as MessageRace>::SourceHeaderId, <P a
 /// State of race target client.
 type TargetClientState<P> = ClientState<<P as MessageRace>::TargetHeaderId, <P as MessageRace>::SourceHeaderId>;
 
+/// Nonces on the race client.
+#[derive(Debug, Clone)]
+pub struct ClientNonces<MessageNonce> {
+	/// Latest nonce that is known to the client.
+	pub latest_nonce: MessageNonce,
+	/// Latest nonce that is confirmed to the bridged client. This nonce only makes
```

Comment:
Right now it is only `Some` in message delivery race.

Reply:
Then, if we'll ever add another race, this comment would need to be updated. I doubt that going through all code comments every time you change something is ok. So I'd like to avoid any race-specific comments in race-agnostic structs, functions, etc